### PR TITLE
execute setup.sh -u if there is a need to

### PR DESCRIPTION
**Pull request by rawgni**
## Short Description:

Instead of telling the user they should execute ./setup.sh -u, we did it for them.

## Fixes/Resolves/Closes (please use correct syntax):
- 
-
-

### CODE REVIEW

**Comment by PokemonGoZ** [comment]
I never executed this when I installed. Explains why my bot never worked unless I did 3,000 different things to do it. Thanks setup.sh

**Comment by DBa2016** [comment]
Please introduce an additional parameter then to PREVENT this behavior. When testing a pull request, having a stable not-changing environment is very helpful.

**Comment by mjmadsen** [comment]
I'll agree with @DBa2016.

**Comment by rawgni** [comment]
@mjmadsen @DBa2016 so now it prompt for user input. will this be better ?

File: run.sh [hunk 1]
```diff
@@ -3,38 +3,39 @@ pokebotpath=$(cd "$(dirname "$0")"; pwd)
...
+if [ "1" == $(git branch -vv |grep -c "* dev") ] && [ $(git log --pretty=format:"%h" -1) != $(git log --pretty=format:"%h" -1 origin/dev) ] ||
+  [ "1" == $(git branch -vv |grep -c "* master") ] && [ $(git log --pretty=format:"%h" -1) != $(git log --pretty=format:"%h" -1 origin/master) ]
 then
-echo "Branch dev have an update. Run ./setup.sh -u to update."
-sleep 2
-elif [ "1" == $(git branch -vv |grep -c "* master") ] && [ $(git log --pretty=format:"%h" -1) != $(git log --pretty=format:"%h" -1 origin/master) ]
-then
-echo "Branch master have an update. Run ./setup.sh -u to update."
-sleep 2
+  read -p "Branch have an update. Run ./setup.sh -u to update? y/n
```
Comment (mjmadsen) [inline run.sh]: has* (know it was in original) but then yes!

**Comment by mjmadsen** [comment]
Never mind my edits. It's better. :)

**Pull request merged**